The '"' confuses the gtk-doc scanner or something. Let's default to the
default log domain for now.
--- libnotify-0.7.8/libnotify/meson.build.orig	2019-06-05 18:18:44.331667900 +0200
+++ libnotify-0.7.8/libnotify/meson.build	2019-06-05 18:19:05.565092100 +0200
@@ -33,7 +33,7 @@
 )
 
 libnotify_cflags = [
-  '-DG_LOG_DOMAIN="libnotify"',
+# '-DG_LOG_DOMAIN="libnotify"',
 ]
 
 mapfile = meson.project_name() + '.map'
